Main.java

```diff
@@ -667,10 +679,15 @@ private static void finishFight(Client winner, Client loser) {
       winner.giveItem(Game.Item.HPOTION);
       msg(winner, "You found 1 healing potion!");
     } else {
-      int rnd = Utils.rndInRange(1,4);
+      // logic for looting bots is here
+      int rnd = Utils.rndInRange(1,6);
       if (rnd == 1) {
         winner.giveItem(Game.Item.HPOTION);
         msg(winner, "You found 1 healing potion!");
+      } else if (rnd < 4) {
```

Comment:
So there is about 50% chance to get some item drop?

Reply:
1 out of 6 to get health potion,
2 out of 6 or 1/3 to get an item.
1/6 + 1/3 = 3 / 6 = 1 / 2.

Yes, looks like you're right :) Do you think we should change that?